src/mad_aper.c

```diff
@@ -560,7 +560,7 @@ else if (!strcmp(apertype,"racetrack")) {
     // zero horizontal or vertical explosion factors, and zero radius should be allowed.
     // 2015-Mar-09  14:43:33  ghislain: change meaning of parameters: ap1 and ap2 are now full rectangle extension,
     // 2015-Mar-10  10:18:46  ghislain: change rounded corners from circular to generalized elliptical shape
-  if ( (*ap1) < 0 || (*ap2) < 0 || (*ap3) < 0 || (*ap4) < 0 || (*ap1) < (*ap3) || (*ap2) < (*ap4)) {
+  if ( (*ap1) < 0 || (*ap2) < 0 || (*ap3) <= 0 || (*ap4) < 0 || (*ap1) < (*ap3) || (*ap2) < (*ap4)) {
```

Comment:
perhaps `(*ap4) <= 0` would be needed as well?

Reply:
I think you are right. Will fix that in another PR.  Thanks!